packages/layer-composer/src/generators/heatmap/heatmap-animated.ts

```diff
@@ -208,6 +209,10 @@ class HeatmapAnimatedGenerator {
       finalConfig.interval
     )
 
+    if (timeChunks.deltaInIntervalUnits === 0 && config.aggregationOperation !== AggregationOperation.Avg) {
+      throw new Error('Trying to show less than 1 interval worth of data for this heatmap layer. This could result in showing misleading information when aggregation mode is not set to avg.')
```

Comment:
Throwing a new error might cause an ~white~ error boundaries screen... why not use console.warn or error instead ? or is there already a place to catch that error ?